readme.txt

```diff
@@ -259,6 +259,10 @@ To apply these changes, you must go to Settings > Permalinks and click "Save Cha
 
 == Changelog ==
 
+= 3.14.4 =
```

Comment:
i think this is supposed to be 3.14.5 :)

Reply:
Yes, corrected